sdk-rs/src/dlob.rs

```diff
@@ -112,6 +119,86 @@ impl DLOBClient {
 
         RxStream(rx)
     }
+
+    /// Subscribe to an orderbook via WebSocket.
+    /// You MUST have a Drift Client initialized to use this fn.
+    pub async fn subscribe_ws(&self, market: MarketId) -> SdkResult<L2OrderbookStream> {
+        env_logger::init();
```

Comment:
this should go at the top level of an app not here. could put it in the test case if want to view the output